plugins/source/onfleet/client/spec.go

```diff
@@ -0,0 +1,5 @@
+package client
+
+type Spec struct {
+	// plugin spec goes here
```

Comment:
No API key option in the spec? Only Env vars?

Reply:
Putting plaintext API keys in plaintext files always seems not-great to me :) We also don't allow it in our big plugins AFAIK (aws, gcp
,...). 
It's a config option I'm happy to omit.

Reply:
I.e. I think this feature can wait until a cutsomer requests it (hopefully they won't).